docs/devGuide/bootcamp/exploreMarkBind.md

```diff
@@ -0,0 +1,242 @@
+{% set title = "Explore MarkBind as a User" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: 3
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+Getting to know MarkBind as a user will help you understand what MarkBind offers and how it can be used to create static websites. This will also help you understand the different components and syntax available in MarkBind, which will be useful when you want to debug issues or create similar features in the future.
+
+</div>
+
+## TLDR
+
+- [ ] Setup the master branch MarkBind
+- [ ] Create a MarkBind site
+- [ ] Create content with MarkBind
+- [ ] Modify site structure and configuration
+- [ ] Deploy your MarkBind site
+
+## Setup the master branch MarkBind
+
+As mentioned in our [user guide]({{baseUrl}}/userGuide/gettingStarted.html), we can either install MarkBind via npm or create a new MarkBind site with npx.
+
+In this bootcamp, we want to set up the master branch MarkBind so that we can test out any changes we make to the codebase on our local machine.
+
+<box type="tip" light>
+
+You can also switch between the master branch MarkBind and check out other branches if you are working on multiple issues. This is also useful when you want to test out a PR submitted by another developer.
+
+</box>
+
+<panel src="{{baseUrl}}/devGuide/development/settingUp.md" header="**Setting up MarkBind**" type="info" minimized></panel>
+
+<box type="important" light>
+
+**Checkpoint**
+
+After setting up MarkBind, try running from the root directory of your MarkBind repository:
+- [ ] `markbind -v` to check that you have the correct version of MarkBind installed
+- [ ] `cd docs && markbind serve -d` to check that you can serve the MarkBind documentation site in development mode
+
+If you can browse the MarkBind documentation site at http://localhost:8080, you have successfully set up MarkBind!
```

Comment:
Very minor nit: I believe if port 8080 is already in use (maybe some other server is using it on the user's computer), MarkBind will be served from another port. 

The URL address should be shown in the CLI output so perhaps we can mention this instead?